src/DefaultNodeTypes.jl

```diff
@@ -2,6 +2,9 @@
 
 SamplableBelief = Union{Distributions.Distribution, KernelDensityEstimate.BallTreeDensity, AliasingScalarSampler}
 
+#Supported types for parametric
+ParametricTypes = Union{Normal, MvNormal}
```

Comment:
think we should only use `MvNormal` in all cases.

Reply:
I'll leave it as is for now to fit in with mm that currently works with `Normal` and `MvNormal`. We can change it on both at a later stage if you want.

Reply:
okay